docs/release-notes/release-notes-0.6.3.1.md

```diff
@@ -0,0 +1,228 @@
+
+`6d87322` Remove jm_single config var call from jmdaemon
+
+### CLI improvements and bugfixes
+
+This includes a change in default minimum size for yieldgenerators, the ability to query software version with the `--version` flag, and an occasionally important improvement in tumbler behaviour: sometimes attempts to broadcast transactions fail due to 'mempool conflict' (one of the inputs provided by a maker, is already spent). But the algorithm was treating this case very wrongly: it was assuming that the cause of the stalling was that some counterparty responded wrongly (e.g. not sending a signature), and so was re-attempting the transaction with all counterparties who *did* respond; this basically ensured repeated failure, because whatever input was already spent, is of course, always already spent. Instead after this fix, the bot will try again with new counterparties.
+
+`3876c1c` Change shebang to python3 and +x for scripts
+
+`49ea4e4` Correct display of wallet mnemonic extension in cli
+
+`c8d10e5` Add python shebangs to joinmarketd.py and wallet-tool.py
+
+`e5cf95d` Add --version option to cli scripts
+
+`6fa7f4c` Display address for UTXO on wallet-tool.py freeze
+
+`4af9c42` Don't use honest_makers in case: mempool conflict
+
+`d2a1a9c` yg-privacyenhanced: change default cj min size to 0.001 btc
+
+### Quit gracefully if connection to Bitcoin Core fails
+
+Previous to this change, we could get a crash and hang of the process in some cases, if the RPC connection to Core failed, such as when Bitcoin Core stops running, or a network disconnect. After these changes we at least get a clear error message and a shutdown of the Joinmarket bot. In the future, this might be extended to support having the Joinmarket program simply wait until the connection comes back up, but there are some
+open questions around that.
+
+`52b9ba8` Initialization of BCI raises Exception not sys.exit
+
+`cc219cc` Handles RPC failure in Qt with message box and quit.
+
+`5ce4b55` Quit gracefully if Bitcoin RPC connection lost
+
+`0c73074` Close wallet when WalletService is stopped.
+
+### Support limited use of scripts and Qt without access to Bitcoin Core
+
+In some cases when an RPC connection to a synced instance of Bitcoin Core is not available, it may be useful for users to be able to see, for example, wallet addresses, or other wallet info such as the seed. This is now supported by setting the `blockchain_source` field of the `BLOCKCHAIN` section to `no-blockchain`. (Note: this option existed already, but it previous to this change only allowed the running of the `ob-watcher.py` script; now it also allows these other things).
+
+`f26186d` Allow accessing basic wallet info without blockchain source
+
+### BIP21 support
+
+This allows users to use BIP21 URLs instead of addresses both in the command line and JoinmarketQt.
```

Comment:
Yes (I thought I was saying that ("and JoinmarketQt"), but it can be more explicit). Also this should be given more prominence. I will slightly reorder the doc for that.